src/platforms/javascript/common/session-replay/troubleshooting.mdx

```diff
@@ -62,6 +62,20 @@ Because of the complexity of the browser environment, there's a significant amou
 
 We're always working on ways to reduce the bundle size. Additionally, there are steps you can take in order to reduce the size of Session Replay based on your specific use case. See [Tree Shaking](../../configuration/tree-shaking/) for more information.
 
+## How does Rage Click detection work?
+
+The Replay SDK tries to detect cases where a user clicked on something, but nothing happened. This is called a "Dead Click", a click that seems to be "dead" and does nothing, from the users perspective. This can happen if a bug or similar prevents the UI from being correctly updated after a button is clicked.
+
+The way the Replay SDK detects this is by checking if a click on a `<button>` or `<a>` element does not lead to any updates of the DOM, or a page scroll. If within 7 seconds no updates are detected, the SDK will generate a "Slow Click" in the Replay UI. We call it a "Slow Click" because it may be possible that something will happen after the 7s, but we chose this as a cutoff time.
```

Comment:
```suggestion
The way that the Replay SDK detects this is by checking if a click on a `<button>` or `<a>` element does not lead to either updates to the DOM or a page scroll. If within 7 seconds no updates are detected, then the SDK will generate a "Slow Click" in the Replay UI. We call it a "Slow Click" because it may be possible that something will happen after the 7s, but we chose this as a cutoff time.
```